package/src/components/Message/MessageSimple/MessageSimple.tsx

```diff
@@ -215,9 +218,13 @@ const MessageSimpleWithContext = (props: MessageSimplePropsWithContext) => {
     shouldRenderSwipeableWrapper,
   );
 
-  const onSwipeToReply = useCallback(() => {
+  const onSwipeActionHandler = useCallback(() => {
```

Comment:
You can do `useStableCallback` here, since this is an actionable method. It'll be safe to use and it'll keep the callback stable as a dependency later on.

Reply:
Okay